src/trade.c

```diff
@@ -2004,62 +2076,57 @@ static void DrawTradeMenuPartyMonInfo(u8 whichParty, u8 monIdx, u8 x, u8 y, u8 w
             break;
         }
     }
-    sTradeMenuData->tilemapBuffer[(y - 1) * 32 + x + 1] = symbolTile;
+    sTradeMenu->tilemapBuffer[(y - 1) * 32 + x + 1] = symbolTile;
 }
 
-static void DrawTradeMenuPartyInfo(u8 whichParty)
+static void PrintPartyLevelsAndGenders(u8 whichParty)
 {
     s32 i;
-    for (i = 0; i < sTradeMenuData->partyCounts[whichParty]; i++)
+    for (i = 0; i < sTradeMenu->partyCounts[whichParty]; i++)
     {
-        const u8 (*r5)[2];
-        const u8 (*r4)[2];
-        u32 r0 = 3 * whichParty;
-        const u8 (*r1)[2][2] = sTradeMonLevelCoords;
-
-        r5 = r1[r0];
-        r4 = sTradeMonBoxCoords[r0];
-        DrawTradeMenuPartyMonInfo(
+        s32 j = i + PARTY_SIZE * whichParty;
+        PrintLevelAndGender(
             whichParty,
             i,
-            r5[i][0],
-            r5[i][1],
-            r4[i][0],
-            r4[i][1]
+            sTradeMonLevelCoords[j][0],
+            sTradeMonLevelCoords[j][1],
+            sTradeMonBoxCoords[j][0],
+            sTradeMonBoxCoords[j][1]
         );
     }
 }
 
-static void ResetTradeMenuPartyPositions(u8 whichParty)
+static void ShowTradePartyMonIcons(u8 whichParty)
 {
     int i;
 
-    for (i = 0; i < sTradeMenuData->partyCounts[whichParty]; i++)
+    for (i = 0; i < sTradeMenu->partyCounts[whichParty]; i++)
     {
-        gSprites[sTradeMenuData->partySpriteIds[whichParty][i]].invisible = FALSE;
-        gSprites[sTradeMenuData->partySpriteIds[whichParty][i]].x = sTradeMonSpriteCoords[(whichParty * PARTY_SIZE) + i][0] * 8 + 14;
-        gSprites[sTradeMenuData->partySpriteIds[whichParty][i]].y = sTradeMonSpriteCoords[(whichParty * PARTY_SIZE) + i][1] * 8 - 12;
-        gSprites[sTradeMenuData->partySpriteIds[whichParty][i]].x2 = 0;
-        gSprites[sTradeMenuData->partySpriteIds[whichParty][i]].y2 = 0;
+        gSprites[sTradeMenu->partySpriteIds[whichParty][i]].invisible = FALSE;
+        gSprites[sTradeMenu->partySpriteIds[whichParty][i]].x = sTradeMonSpriteCoords[(whichParty * PARTY_SIZE) + i][0] * 8 + 14;
+        gSprites[sTradeMenu->partySpriteIds[whichParty][i]].y = sTradeMonSpriteCoords[(whichParty * PARTY_SIZE) + i][1] * 8 - 12;
+        gSprites[sTradeMenu->partySpriteIds[whichParty][i]].x2 = 0;
+        gSprites[sTradeMenu->partySpriteIds[whichParty][i]].y2 = 0;
     }
 }
 
-static void PrintNicknamesForTradeMenu(void)
+static void PrintTradePartnerPartyNicknames(void)
 {
     rbox_fill_rectangle(1);
-  //PrintPartyNicknamesForTradeMenu(TRADE_PLAYER); ?
-    PrintPartyNicknamesForTradeMenu(TRADE_PARTNER);
+  //PrintPartyNicknames(TRADE_PLAYER); ?
```

Comment:
What does this comment mean?

Reply:
That's pre-existing. But presumably wondering if the commented line should be there

Reply:
Kinda looks like a bugfix suggestion